### ISSUE TITLE
D-VQA

### ISSUE THREAD

**Issue by christophschuhmann** [LAION-AI/dataset-tasks#22]
https://paperswithcode.com/dataset/dvqa

**Comment by CipherVector** [comment]
Hi, I started processing this db

**Comment by CipherVector** [comment]
Just confirm
you would have the image file and the metadata would be in json right?

**Comment by christophschuhmann** [comment]
yes, and a "txt" field with the caption, like stated here: https://github.com/LAION-AI/dataset-spec

**Comment by christophschuhmann** [comment]
any updates? :)
